docs/release-notes.md

```diff
@@ -1,45 +1,41 @@
+--8<-- "References/abbreviations.md"
 # Release Notes
 
 If you haven't had a chance - checkout the [Letter From Devs](./letter.md) for a message from our development team.
-## Key Feature Changes for 1.2
-This section outlines key features changes between versions 1.1 and 1.2 for [Open Active Management Technology (Open AMT) Cloud Toolkit.](Glossary.md#o) 
+## Key Feature Changes for 1.3
+This section outlines key features changes between versions 1.2 and 1.3 for Open AMT Cloud Toolkit. 
 
 ### Additions
-#### [RPS](Glossary.md#r)
-- **Hostname and Device Metadata tags:** We have added the ability to define metadata tags for a each Intel&reg; AMT profile.  These tags will be applied to each device that is configured using the profile.  Additionally, if RPC sends up the Intel&reg; AMT device OS hostname during activation, RPS will save this information as part of the device information.  When RPS configures a device, it will perform a POST call to MPS to register that device in MPS.  This works in conjunction with the improved metadata gathering that RPC added in 1.1.
-- **Node 14 LTS support:** RPS has been updated to support running on Node 14 LTS. 
+#### RPS & MPS
+- **Stateless Authentication:** The RESTful APIs for both RPS and MPS now utilizie stateless authentication tokens to validate user access.  With this change, our reference deployment now utilizes Kong API Proxy that validates the token and then fowards authenticated requests to either MPS or RPS.  This simplifies how API calls are made into the Open AMT Cloud Toolkit services by putting the service name that is being called into the URL.  
```

Comment:
utilizie => utilize

Reply:
The first sentence I can't get past. You clarify in the second sentence that KONG is doing the validation rendering the first sentence false. So I'd  say something like , the Restful APIs for both RPS and MPS have had their APIKEYS removed and now leverage KONG for authentication with JWT Tokens.  -- Anywhere you say token i'd say JWT token specifically.

Reply:
Yeah, I struggled here as well. I tried to rewrite in the comment, but I fear I may have changed the meaning (or not).

Reply:
Rephrased this section